### ISSUE TITLE
Show minutes left only if the stage is set to Boil

### ISSUE THREAD

**Issue by icyflame** [OpenBrews/OpenBrews#38]
The `ng-if` directive will come in handy, use something like:

``` html
<label ... ng-if="other.stage.toLowerCase() === 'boil'">... Minutes Left</label>
```

This will ensure that the label element will show up only if the stage is set to `boil`.

**Comment by jaimellamasi** [comment]
@icyflame I'll do it ;)

**Comment by MichaelWashburnJr** [comment]
Closed in #45
